src/dinterpret.d

```diff
@@ -4909,6 +4909,33 @@ public:
             fd = (cast(VarExp)ecall).var.isFuncDeclaration();
             assert(fd);
 
+            if (fd.ident == Identifier.idPool("__ctfeWrite"))
+            {
+                foreach(arg;*e.arguments)
```

Comment:
If you do `__ctfeWriteln("Hello"," ","World",17);`
It will write "Hello World" to the output and only after that exit with an error. 
I'd like to avoid this since it looks confusing.